kornia/models/segmentor/segmentation_models.py

```diff
@@ -47,27 +51,30 @@ def __init__(
         )
 
     def preprocessing(self, input: Tensor) -> Tensor:
-        if self.preproc_params["input_space"] == "RGB":
+        # Ensure the color space transformation is ONNX-friendly
+        input_space = self.preproc_params["input_space"]
+        input = kornia.color.rgb_to_bgr(input) if input_space == "BGR" else input  # Assume input is already RGB if not BGR
```

Comment:
For internal development we are usually importing as `from Kornia.color.rgb import rgb_to_bgr` to avoid conflicts with the namespace